app/ide-desktop/lib/content/src/serviceWorker.ts

```diff
@@ -0,0 +1,28 @@
+/** @file A service worker that redirects paths without extensions to `/index.html`.
```

Comment:
Can we add explanation here why we need it? Something like `Because we use virtual router every time we refresh page with location different than / react throws 404. We need to manually check for routes and navigate to index.html`